cmd/dlv/cmds/commands.go

```diff
@@ -86,11 +88,12 @@ func New(docCall bool) *cobra.Command {
 	}
 
 	RootCommand.PersistentFlags().StringVarP(&Addr, "listen", "l", "localhost:0", "Debugging server listen address.")
-	RootCommand.PersistentFlags().StringVarP(&Log, "log", "", "false", `Comma separated list of components that should produce debug output, possible values:
+	RootCommand.PersistentFlags().BoolVarP(&Log, "log", "", false, "Enable debugging server logging.")
+	RootCommand.PersistentFlags().StringVarP(&LogOutput, "log-output", "", "false", `Comma separated list of components that should produce debug output, possible values:
```

Comment:
Any reason to default to "false" instead of just empty string? Also, since we have both flags we should force users to provide both `--log` to enable logging and then use the other flag to then control the output. It seems that's how it works implicitly now, but would be good to print an error or something if the flags are used incorrectly.

Reply:
It was like this because of the backwards compatibility when it was still called `--log`, you are right and I have changed it.